### PR TITLE
add project smartRE 

### PR DESCRIPTION
**Pull request by VladimirKuzlo**
#288

### CODE REVIEW

**Comment by akhavr** [comment]
please, fix the conflicts

File: projects/smartre.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Year founded: 2017  
+Currency: SRE  
+ICO: Q4 2017
+## Staff
+Co-founder & CEO: [Lloyd Huang](../people/lloyd_huang.md)  
+Co-founder & CTO: [Thomas Phan](../people/thomas_phan.md)  
+CMO China: [H.L. Lu](../people/h.l._lu.md)  
+## Business Model
+SmartRE is a highly secured, decentralized and democratized platform
+whereby US homeowners can liquidated a percentage of their equity in their
```
Comment (akhavr) [inline projects/smartre.md]: liquidated -> liquidate ?

**Comment by VladimirKuzlo** [comment]
fixed

**Comment by akhavr** [comment]
Closes #288

**Pull request merged**